packages/iov-bns/src/bnsconnection.ts

```diff
@@ -620,8 +622,19 @@ export class BnsConnection implements BcpAtomicSwapConnection {
     return nfts;
   }
 
-  public async getFeeQuote(): Promise<Fee> {
-    throw new Error("Not implemented");
+  public async getFeeQuote(transaction: UnsignedTransaction): Promise<Fee> {
```

Comment:
Yes, good enough filler for now.
I will add real logic here when I add weave 0.12 support

Reply:
Jap. This current explicit 0 fee logic should work for the wallet/hugnet and shows the path for future changes.